added sphinx documentation (#64)
thegraphnetwork/epigraphhub_py: docs/source/overview.md

# Overview

> The epigraphhub library is designed to provide for external users all the analytical functionality used in powering the EpigraphHub platform and applications. In fact all application designed to run on the EpigraphHub platform rely strongly on it.

Reviewer: ```suggestion
The epigraphhub library is designed to provide external users with all of the analytical functionality used to power the EpigraphHub platform and applications. In fact, all applications designed to run on the EpigraphHub platform rely heavily on it.
```

> This library should be used by users that want to contribute to the platform as well as by users that want the borrow some of the tools developed by our team in other opensource projects.

Reviewer: ```suggestion
This library should be used by users that want to contribute to the platform, as well as users who want the borrow some of the tools developed by our team in other open source projects.
```

> This library should be used by users that want to contribute to the platform as well as by users that want the borrow some of the tools developed by our team in other opensource projects.

Reviewer 2: ```suggestion
This library should be used by users who want to contribute to the platform and users who want to borrow some of the tools developed by our team in other open-source projects.
```
